src/serac/physics/operators/odes.cpp

```diff
@@ -42,10 +42,73 @@ void SecondOrderODE::SetTimestepper(const serac::TimestepMethod timestepper)
     case serac::TimestepMethod::FoxGoodwin:
       ode_solver_ = std::make_unique<mfem::FoxGoodwinSolver>();
       break;
+    case serac::TimestepMethod::NewmarkBeta:
+      ode_solver_ = std::make_unique<mfem::NewmarkSolver>();
+      break;
+    case serac::TimestepMethod::BackwardEuler:
+      ode_system_solver_ = std::make_unique<mfem::BackwardEulerSolver>();
+      break;
     default:
       SLIC_ERROR("Timestep method was not a supported second-order ODE method");
   }
-  ode_solver_->Init(*this);
+
+  if (ode_solver_) ode_solver_->Init(*this);
+
+  if (ode_system_solver_) {
+    // we need to adjust the width of this operator
+    width *= 2;
+    ode_system_solver_->Init(*this);
+  }
+}
+
+void SecondOrderODE::Step(mfem::Vector& x, mfem::Vector& dxdt, double& t, double& dt)
+{
+  if (ode_solver_) {  // if we used a 2nd order method
```

Comment:
Do we ever have both an `ode_system_solver` and an `ode_solver`?

Reply:
I have not run into a case like that yet. I think `SecondODESolver` and `OdeSolver` are completely separate, so usually they would never interact.